### DISCUSSION TITLE
PGID to high according to Gluetun

### DISCUSSION THREAD

**Discussion by Deja024** [qdm12/gluetun#973] [Q&A] [answered]
Hi, I am new here and have been searching the discussion forums for solutions but without any luck. But if I put a repeat question here then just say it and I in that case I apologize for being a rookie.

I am installing the gluetun docker on my Synology 920+ DSM 7.01. All dockers that need a PUID and PGID are pointed to a special user and group created for docker use. (UID 1033 Dockeruser and GID 65539 Dockergroup) The UID- and GID-number are handed out by DSM when creating them. 

Now I never had problems running containers with these but the Gluetun container terminates saying: "ERROR reading from environment variables: environment variable PGID: system ID is not valid: 65539: must be between 0 and 65535". I would love to comply but Synology starts handing out GIDs above that number and for what I learned from multiple synology forums, changing those is not possible.

So my question, is it possible to make Gluetun accept PGIDs above 65535??

**Comment by qdm12** [comment, marked as answer]
Hi there,

I wrongly assumed the uid and gid in Linux was limited to be an unsigned 16 bit integer (up to 65535), but it looks like most Linux kernels now accept 32 bit unsigned integers.

This is changed on the latest image in 9dd5e7bf1ddf93e82e7ab36ac9e41d266dd23c8b to accept 32 bit unsigned integers (so larger) :wink:

**Reply by Deja024** [reply]
Wow, thanks for this fast response. Will try it out as soon as I can!
